README.md

```diff
@@ -1,2 +1,153 @@
 # ilib-loctool-mrkdwn
-A loctool plugin that can parse Slack mrkdwn messages in a json file
+
+Ilib loctool plugin to parse and localize the slack-flavored markdown
+known as "mrkdwn" encoded in js files.
+
+## File Format
+
+A slack application can encode its strings as a simple js file with
+the properties being the unique ids of the strings, and the values
+being strings in mrkdwn format.
+
+Example file:
+
+```javascript
```

Comment:
Inline code comments are missing in this example

Reply:
I'll add them